js/modules/k6/experimental/tracing/trace_id.go

```diff
@@ -0,0 +1,76 @@
+package tracing
+
+import (
+	"crypto/rand"
+	"encoding/binary"
+	"encoding/hex"
+	"fmt"
+)
+
+const (
+	// Being 075 the ASCII code for 'K' :)
+	k6Prefix = 0o756
+
+	// To ingest and process the related spans in k6 Cloud.
+	k6CloudCode = 12
+
+	// To not ingest and process the related spans, b/c they are part of a non-cloud run.
+	k6LocalCode = 33
+)
+
+// TraceID represents a trace-id as defined by the [W3c specification], and
+// used by w3c, b3 and jaeger propagators. See [Considerations for trace-id field generation]
+// for more information.
+//
+// [W3c specification]: https://www.w3.org/TR/trace-context/#trace-id
+// [Considerations for trace-id field generation]: https://www.w3.org/TR/trace-context/#considerations-for-trace-id-field-generation
+//
+//nolint:lll
+type TraceID struct {
+	Prefix            int16
+	Code              int8
+	UnixTimestampNano uint64
+}
+
+// NewTraceID returns a new TraceID with the given prefix, code and unix timestamp in nanoseconds.
+func NewTraceID(prefix int16, code int8, unixTimestampNano uint64) TraceID {
+	return TraceID{
+		Prefix:            prefix,
+		Code:              code,
+		UnixTimestampNano: unixTimestampNano,
+	}
+}
+
+// Encode encodes the TraceID into a hex string and a byte slice.
```

Comment:
Why does it return both? If it has a reason good to have in the comment.

Reply:
None indeed, I blindly reproduced the behavior from xk6-distributed-tracing, but it's not necessary indeed 👍🏻 Removing it ✂️